src/Rules/NotIn.php

```diff
@@ -45,6 +45,16 @@ public function check(mixed $value): bool
     {
         $this->assertHasRequiredParameters(['disallowed_values']);
 
+        if (is_array($value)) {
```

Comment:
Based on the comments from In, I am not sure about combining this into the NotIn rule.